content/docs/command-reference/remote/modify.md

```diff
@@ -444,6 +444,11 @@ order of precedence):
 
 Note that Azure remotes can also authenticate via environment variables (instead
 of `dvc remote modify`). These are tried if none of the params above are set.
+Also some options can be propagated from the azure config file (which is managed
+through
+[az config](https://docs.microsoft.com/en-us/cli/azure/config?view=azure-cli-latest)).
+The default directory where the config will be searched is `~/.azure` but it can
```

Comment:
Questions:
- Which options can come from an Azure config file?
- Will remote config params take precedence over Azure config file options?

Reply:
> Which options can come from an Azure config file?

`connection_string`, `account_name`, `account_key`, `sas_token` and `container_name

> Will remote config params take precedence over Azure config file settings?
> Will env vars take precedence over Azure config file settings?

Yes, and yes. Precedence order: `dvc config > environment variables > azure config`

Reply:
Thanks for the info. I'll take it from here 👍